Update dag-run.rst (#27470)
mistercrunch/Airflow: docs/apache-airflow/dag-run.rst

=========
A DAG Run is an object representing an instantiation of the DAG in time.
Any time the DAG is executed, a DAG Run is created and all tasks inside it are executed. The status of the DAG Run depends on the tasks states.

> Each DAG Run is run separately from another, meaning that you can have many running DAGs at the same time.

Reviewer: ```suggestion
Each DAG Run is run separately from one another, meaning that you can have many runs of a DAG at the same time.
```

I think this is what the original was trying to say.
